lib/const.rb

```diff
@@ -1,6 +1,10 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.28.4'
+CREW_VERSION = '1.28.5'
+
+# This is not a constant, but may be set once on every crew run from
+# package.rb, based upon which package is first being operated upon.
+@crew_current_package = nil
```

Comment:
Is it even necessary to set here?  Isn't a variable set to nil by default?

Reply:
You are right. Removed that.